### PR TITLE
Dark mode implementation for AdministratorControls, Help, Options, ProfileReset, ProfileRename, AppVersion, ProfileList, ProfileEdit etc

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR fixes dark mode for following activities:

**OptionsActivity**
- ReadingTextSizeActivity
- AudioLanguageActivity
- AppLanguageActivity

**HelpActivity**
- FAQListActivity
- FAQSingleActivity
- ThirdPartyDependencyActivity
- LicenseListActivity
- LicenseViewerActivity
- PolicyViewerActivity

**AdministratorControlsActivity**
- AppVersionActivity
- ProfilListActivity
- ProfilEditActivity
- ProfileRenameActivity
- ProfileResetPinActivity

There were a lot of issues with mocks, for example:
- Mocks were not available for reset, rename screens in light-mode portrait mobile
- Dark mode mocks were not available for some internal screens
- Dark mode mocks were not available for tablet multi-pane screens.
- There were inconsistencies in designs of similar screen.

This PR makes sure that all the screens support dark-mode and also the components across various screens should have consistent design guidelines.

## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### Options Activity and its internal screens
| Before  | After |
| ------------- | ------------- |
| Screenshot_20221221_113351 | Screenshot_20221221_113418 |
| Screenshot_20221221_113357 | Screenshot_20221221_113424 |
| Screenshot_20221221_113403 | Screenshot_20221221_113431 |
| Screenshot_20221221_113836 | Screenshot_20221221_113853 |
| Screenshot_20221221_113842 | Screenshot_20221221_113859 |

### Help Activity and its internal screens
| Before  | After |
| ------------- | ------------- |
| Screenshot_20221221_114027 | Screenshot_20221221_114152 |
| Screenshot_20221221_114034 | Screenshot_20221221_114157 |
| Screenshot_20221221_114039 | Screenshot_20221221_114203 |
| Screenshot_20221221_114045 | Screenshot_20221221_114210 |
| Screenshot_20221221_114051 | Screenshot_20221221_114214 |
| Screenshot_20221221_114100 | Screenshot_20221221_114221 |
| Screenshot_20221221_114110 | Screenshot_20221221_114227 |
| Screenshot_20221221_114114 | Screenshot_20221221_114234 |
| Screenshot_20221221_114615 | Screenshot_20221221_114724 |
| Screenshot_20221221_114621 | Screenshot_20221221_114728 |
| Screenshot_20221221_114627 | Screenshot_20221221_114731 |
| Screenshot_20221221_114632 | Screenshot_20221221_114736 |
| Screenshot_20221221_114641 | Screenshot_20221221_114741
| Screenshot_20221221_114649 | Screenshot_20221221_114749 |
| Screenshot_20221221_114656 | Screenshot_20221221_114754 |

### Administrator Controls and its internal activities

| Before  | After |
| ------------- | ------------- |
| Screenshot_20221221_115905 | Screenshot_20221221_115954 |
| Screenshot_20221221_115912 | Screenshot_20221221_120004 |
| Screenshot_20221221_115919 |Screenshot_20221221_120009 |
| Screenshot_20221221_120012 | Screenshot_20221221_115925 |
| Screenshot_20221221_115929 | Screenshot_20221221_120018 |
| Screenshot_20221221_120022 | Screenshot_20221221_115934 |
| Screenshot_20221221_120404 | Screenshot_20221221_120449 |
| Screenshot_20221221_120411 | Screenshot_20221221_120455 |
| Screenshot_20221221_120420 | Screenshot_20221221_120500 |
| Screenshot_20221221_120426 | Screenshot_20221221_120505 
| Screenshot_20221221_120433 | Screenshot_20221221_120510 |

### CODE REVIEW

File: app/src/main/res/layout-sw600dp/help_activity.xml [hunk 1]
```diff
@@ -40,9 +40,10 @@
         android:layout_height="48dp"
         android:layout_marginStart="12dp"
         android:layout_marginTop="12dp"
-        android:background="@color/component_color_help_activity_background_color"
+        android:background="@color/component_color_shared_transparent_background_color"
         android:contentDescription="@string/help_activity_back_arrow_description"
         android:padding="12dp"
+        android:tint="@color/component_color_shared_multipane_icon_color"
```
Comment (MohitGupta121) [inline app/src/main/res/layout-sw600dp/help_activity.xml:46]: @rt4914 Please make similar like as shared text color.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@rt4914 PTAL

**Comment by rt4914** [comment]
Me and @MohitGupta121 did review this PR over a 2 hr call where we went through each and every view, compared it with mocks and if something was changed w.r.t to mocks then if both of us agreed we finalised that and applied that change.

Its not possible to review this PR using mocks as they are inconsistent and this PR does require a lot of context.

File: app/src/main/res/layout-sw600dp/help_activity.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-sw600dp/help_activity.xml:46]: Done.

**Review by MohitGupta121 — APPROVED** [review]
@rt4914 LGTM, Thanks.

**Review by MohitGupta121 — APPROVED** [review]
@rt4914 I not find any open issues related to these screens in the [dark-mode-project-board](https://github.com/orgs/oppia/projects/11). So I think there is no need to link any issue.

**Comment by rt4914** [comment]
@seanlip With [reference](https://github.com/oppia/oppia-android/pull/4795#issuecomment-1360921037) to this can you approve this PR?

**Review by seanlip — APPROVED** [review]
Confirming this looks good from a UI perspective. Thanks!

**Pull request merged**